test/backends/xeon/run_test.py

```diff
@@ -0,0 +1,112 @@
+# Owner(s): ["module: unknown"]
```

Comment:
Should this be run in CI?
If so:
- The file should be renamed to be `test_*` so that it actually runs
- We should have a proper module for it. Should I add a `module: xeon` and add you to the cc list for it?
- We need to make sure that it runs on a machine that has all the proper requirements. Do we have any CI machine with the right hardware to run this?

Reply:
- Yeah, sure. I'll rename it to `test_launch_xeon.py`.
- Yeah, sounds good to me.
- 4 out of 5 test functions depend on installation of Jemalloc, TCMalloc and IOMP python packages (`test_jemalloc`, `test_tcmalloc`, `test_iomp` and `test_default_config`). If the installation cannot be done, I'll remove them, but just keep `test_multi_threads` to verify if multiple instances can be launched. This one doesn't require any hardware resources, just a loop to execute a bash command. Do you think if this one is needed? If not, I think we can remove this UT script.

Reply:
cc @malfet who is more familiar with the config on our current CI machines? Do we have the right hardware? And if so, can we install these libraries by default on the runners?